src/main/java/com/github/sardine/ant/command/RecursiveGet.java

```diff
@@ -0,0 +1,118 @@
+package com.github.sardine.ant.command;
+
+import java.io.InputStream;
+import java.net.URI;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.List;
+
+import com.github.sardine.DavResource;
+import com.github.sardine.ant.Command;
+
+/**
+ * A nice ant wrapper around sardine.list() and sardine.get().
+ * 
+ * @author andreafonti
+ */
+public class RecursiveGet extends Command {
+
+	/**
+	 * Webdav server url
+	 */
+	String fServerUrl;
```

Comment:
I'm sorry, i began implementing the change by looking at other code related to ant tasks in order to code in the same style, noticed your comments about prefixes in the other pull requests only after i opened mine.

I will fix the pr in monday.

Thanks for your time